.changeset/heavy-windows-dance.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sap-ux-private/preview-middleware-client': patch
```

Comment:
when a change is made in `preview-middleware-client`, then also a changeset for the `preview-middleware` needs to be added as well 😉 Please add one

Reply:
Thanks. Updated :)

Reply:
thx